test/utils/test_objects.go

```diff
@@ -63,6 +72,17 @@ var ExampleDeployment = &appsv1.Deployment{
 							},
 						},
 					},
+					{
+						Name: "configmap1-optional",
```

Comment:
I'd be tempted to rename this to be consistent with the `secret-optional`
```suggestion
						Name: "configmap-optional",
```